spec/sms_sender_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'sms_sender'
+
```

Comment:
What happens if you break the SmsSender methods in some way – do these tests fail?